eldritch/test_events.py

```diff
@@ -822,6 +822,56 @@ def testCombatChoice(self):
 
 # TODO: add tests for going unconscious/insane during a mythos/encounter.
 
+class CloseLocationTest(EventTest):
+  def testCloseForever(self):
+    place_name = "Woods"
+    self.state.event_stack.append(CloseLocation(place_name))
+    self.resolve_until_done()
+    self.assertTrue(self.state.places[place_name].closed)
+
+  def testCloseWithGateForOneTurn(self):
+    place_name = "Woods"
+    place = self.state.places[place_name]
+    place.gate = self.state.gates.popleft()
+    monster = next(iter(self.state.monsters))
+    monster.place = place
+    self.char.place = place
+    self.state.event_stack.append(CloseLocation(place_name, 1))
+    self.resolve_until_done()
+    self.assertEqual(place.closed_until, self.state.turn_number + 2)
+    self.assertEqual(self.char.place, place)
+    self.assertEqual(monster.place, place)
+    self.assertFalse(place.closed)
+    self.state.event_stack.append(GateCloseAttempt(self.char, place_name))
```

Comment:
Use some blank lines to split this test up into logical chunks. e.g. setup, first round of assertions, closing the gate, second round of assertions